### ISSUE TITLE
API endpoint to delete content DB from file system

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#1495]
In the current implementation on `import-export`, we use the `ChannelResource` to 'delete' a Channel from the server:

```
// manageContentActions.js line 48
ChannelResource.getModel(channelId).delete()
```

This deletes the Channel's metadata, but not the files comprising the Channel. Apparently, this means if the server is restarted, the Channel metadata will be reconstructed and reappear in the app.

Assuming that 'to delete' means to disable the the channel _and_ remove its contents from the device, we need an API to do the later.

An alternative interpretation of 'to delete' is to simply disable the channel but _not_ remove its contents from the device. This would probably also need another API.

**Comment by jonboiser** [comment]
@jamalex To implement this, would it be sufficient to just delete the channel database e.g. `.kolibri/content/databases/channelid.sqlite3`, but not touch the stuff in `.kolibri/content/storage`?

**Comment by jamalex** [comment]
Yes, for now that will be enough. We'll be adding "garbage collection" functionallity at some point, but for now it's better if the content files are left (because they may be referenced by other channel DBs as well).

**Comment by rtibbles** [comment]
Fixed in #1585
